multiapps-common/src/test/java/org/cloudfoundry/multiapps/common/util/MapUtilTest.java

```diff
@@ -12,28 +12,28 @@
 
 class MapUtilTest {
 
-    public static final Map<String, Object> testParameters = new HashMap<String, Object>() {
-        {
-            put("trueFlag", new Boolean(true));
-            put("falseFlag", new Boolean(false));
-            put("emptyFlag", null);
-            put("incorrectTypeFlag1", "false");
-            put("incorrectTypeFlag2", "1");
-        }
-    };
+    public static final Map<String, Object> TEST_PARAMETERS = new HashMap<>();
```

Comment:
If we aren't going to modify this map, why not use the `Map.of(...)` factory constructor?

Reply:
`Map.of(...)` does not accept null values, unfortunately.